react/week4/github-search/src/components/InputForm.js

```diff
@@ -0,0 +1,17 @@
+import React, { useContext } from "react";
+import { ProfileContext } from "./ProfileSearch";
+import DisplayUser from "./DisplayUsers";
+
+function Input() {
+  const { gitHubApiHandler } = useContext(ProfileContext);
+  const onChangeHandler = (event) => {
+    gitHubApiHandler(event.target.value);
```

Comment:
It's a best practice to call side effects like fetch inside the useEffect method. 

I would rather create an input state and pass it into the useEffect dependency array...